lxd/images.go

```diff
@@ -851,6 +851,9 @@ func getImageMetadata(fname string) (*api.ImageMetadata, string, error) {
 
 	// Open the tarball
 	if len(unpacker) > 0 {
+		if algo == ".squashfs" {
```

Comment:
Could do with a comment like `// sqfs2tar can only read from a file` so we remember what this chunk is for :)